client/actions/personalActions.js

```diff
@@ -45,8 +45,6 @@ function fetchThumbnails(idArray, imgArray, bentoData, dispatch, category) {
         type: FETCH_POPULAR_BENTOS,
         payload: bentoData,
       });
-    } else {
```

Comment:
This should be kept for error checking, this shouldn't print out unless something wrong actually happens